kiwipy/communications.py

```diff
@@ -1,14 +1,16 @@
 import abc
 import sys
+from typing import Any
 
 import shortuuid
 
 from . import exceptions
 # For backwards compatibility import exceptions too
 from .exceptions import *  # pylint: disable=wildcard-import, redefined-builtin, unused-wildcard-import
 from . import futures
+from . import types
 
-__all__ = ['Communicator', 'CommunicatorHelper']
+__all__ = 'Communicator', 'CommunicatorHelper'
```

Comment:
```suggestion
__all__ = ('Communicator', 'CommunicatorHelper')
```

Reply:
I'm going by the google style guide here (http://google.github.io/styleguide/pyguide.html#33-parentheses) and that use in python's asyncio library.  If you insist I'll put the tuple parentheses back but I prefer this for single line `__all__`'s.

Reply:
I see, didn't know they maintained that as a style guide for single lines. I prefer to be consistent and just always use a tuple, but I certainly won't insist. Fine to leave as is.